application/lib/disposition.dart

```diff
@@ -39,7 +41,7 @@ class RootDisposition extends ChangeNotifier implements JsonEncodable {
   }
 
   void didChange() {
-    saveFile.save(this);
+    Timer(const Duration(seconds: 1), () => saveFile.save(this));
```

Comment:
you have to remember the Timer in a local variable and cancel it in `dispose` if it's not null

Reply:
also, if the timer is already set, don't set a new one